### DISCUSSION TITLE
An extension that used to work properly does not work.

### DISCUSSION THREAD

**Discussion by Someone-G** [fork-maintainers/iceraven-browser#777] [Q&A]
An extension that used to work properly does not work. I want to install the "Обход блокировок Рунета" extension. It is in the list of available ones. It installs without problems, but it doesn't work! Previously, in previous versions of the browser, it worked correctly. Why? How to fix it?
Iceraven 2.20.0. The smartphone is the same. Androin 8.1.0.

![photo_2024-06-16_12-50-22](https://github.com/fork-maintainers/iceraven-browser/assets/108881628/00cd5f68-3d27-4341-afeb-81a6920ab3f5)  
![Screenshot_20240612-224105](https://github.com/fork-maintainers/iceraven-browser/assets/108881628/2a936646-df4b-4b1f-98bd-0b35381c3ebf)

**Comment by Someone-G** [comment]
I wrote to the extension developer and he replied:
> На Андроиде не поддерживается этот API (proxy.settings): https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/settings. Без него не будет работать.

Translation:
> This API (proxy.settings) is not supported on Android: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/proxy/settings . It won't work without it.

How is it? Everything used to work fine!

**Reply by mzso** [reply]
I think still the extension dev is the best person to answer this.
